src/node/convolver.rs

```diff
@@ -414,6 +411,11 @@ impl ConvolverRendererInner {
     }
 }
 
+struct ConvolverRenderer {
```

Comment:
As said, I will rewrite this.
A pattern of `is_set` and a tombstone value is way better represented as simply `Option<ConvolverRendererInner>`

Reply:
sure this is a redondant